src/utils/FeatureFinderMetaboIdent.cpp

```diff
@@ -259,7 +259,12 @@ class TOPPFeatureFinderMetaboIdent :
     mzml.setLogType(log_type_);
     mzml.getOptions().addMSLevel(1);
     mzml.load(in, ff_mident.getMSData());
-
+    if (ff_mident.getMSData().empty())
```

Comment:
Hmm one could argue that an empty file leads to a successful run.
This might be important in pipeline situations